.github/workflows/bridge.yml

```diff
@@ -0,0 +1,158 @@
+name: tardis-carsus-compatibility-check
```

Comment:
Use a short name here.

Reply:
I was not sure about what name to give to this workflow. We could go with something like `tardis-carsus-bridge` but I felt that was not so descriptive.
Any suggestions?